src/components/Sidebar.jsx

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import Drawer from "@mui/material/Drawer";
+import Toolbar from "@mui/material/Toolbar";
+import List from "@mui/material/List";
+import ListItemButton from "@mui/material/ListItemButton";
+import ListItemIcon from "@mui/material/ListItemIcon";
+import ListItemText from "@mui/material/ListItemText";
+import ExpandLess from '@mui/icons-material/ExpandLess';
+import ExpandMore from '@mui/icons-material/ExpandMore';
+import Collapse from '@mui/material/Collapse';
+import { Button, Divider }  from "@mui/material";
+import LogoutIcon from '@mui/icons-material/Logout';
+import PieChartOutlinedIcon from '@mui/icons-material/PieChartOutlined';
+import GridViewOutlinedIcon from '@mui/icons-material/GridViewOutlined';
+import ReviewsRoundedIcon from '@mui/icons-material/ReviewsRounded';
+import SettingsOutlinedIcon from '@mui/icons-material/SettingsOutlined';
+
+const drawerWidth = 300;
+const logoURL = "https://cdn.discordapp.com/attachments/1198067297517051995/1199890391118381117/blue.JPG?ex=65c43002&is=65b1bb02&hm=1f6fc371eec1deffb670578da5f06d117e0b104ee6d212b8adca6574d7acb003&";
```

Comment:
It's better to download the image file and import from that. All the image files need to be in public folder